tsconfig.json

```diff
@@ -1,7 +1,5 @@
 {
   "compilerOptions": {
-    "rootDir": "./",
-    "baseUrl": ".",
```

Comment:
The changes here in tsconfig make it so the structure of everything inside `types/` matches the structure of everything inside `src/` (i.e. `src/path/foo.js` corresponds to `types/path/foo.d.ts`, previously it was `types/src/path/foo.d.ts`).